.github/workflows/recipe.yaml

```diff
@@ -19,8 +19,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.4]
-        sylius: [1.8, 1.9]
+        php: ['7.4' ,'8.0']
+        sylius: ["~1.8.0", "~1.9.0", "~1.10.0"]
+        exclude:
+          - php: 8.0
+            sylius: ["~1.8.0"]
```

Comment:
```suggestion
            sylius: "~1.8.0"
```

Reply:
my bad, corrected
it's too early, I need a coffee